homeassistant/components/viam/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "viam",
+  "name": "Viam",
+  "codeowners": ["@hipsterbrown"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/viam",
+  "integration_type": "hub",
```

Comment:
hub doesn't seem correct

Reply:
Based on my understanding from [this blog post](https://developers.home-assistant.io/blog/2022/10/24/hubs-devices-services-virtual-integrations?_highlight=integration_type#differentiating-hubs-devices-and-services), this integration provides multiple services for the single config entry. 

I'd like to introduce the ability to expose devices connected to Viam entities in Home Assistant as follow up to this integration.

Reply:
You're right, thanks for linking to the blog post 👍